spec/abci/abci++_methods.md

```diff
@@ -747,42 +748,42 @@ Most of the data structures used in ABCI are shared [common data structures](../
+    | validator | [ValidatorParams](../core/data_structures.md#validatorparams) | Parameters limiting the types of public keys validators can use.             | 3            | Yes           |
+    | version   | [VersionsParams](../core/data_structures.md#versionparams)    | The ABCI application version.                                                | 4            | Yes           |
 
 ### ProofOps
 
 * **Fields**:
 
-    | Name | Type                         | Description                                                                                                                                                                                                                  | Field Number |
-    |------|------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
-    | ops  | repeated [ProofOp](#proofop) | List of chained Merkle proofs, of possibly different types. The Merkle root of one op is the value being proven in the next op. The Merkle root of the final op should equal the ultimate root hash being verified against.. | 1            |
+    | Name | Type                         | Description                                                                                                                                                                                                                  | Field Number | Deterministic |
+    |------|------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|---------------|
+    | ops  | repeated [ProofOp](#proofop) | List of chained Merkle proofs, of possibly different types. The Merkle root of one op is the value being proven in the next op. The Merkle root of the final op should equal the ultimate root hash being verified against.. | 1            | No            |
 
 ### ProofOp
 
 * **Fields**:
 
-    | Name | Type   | Description                                    | Field Number |
-    |------|--------|------------------------------------------------|--------------|
-    | type | string | Type of Merkle proof and how it's encoded.     | 1            |
-    | key  | bytes  | Key in the Merkle tree that this proof is for. | 2            |
-    | data | bytes  | Encoded Merkle proof for the key.              | 3            |
+    | Name | Type   | Description                                    | Field Number | Deterministic |
+    |------|--------|------------------------------------------------|--------------|---------------|
+    | type | string | Type of Merkle proof and how it's encoded.     | 1            | No            |
```

Comment:
Similar comment, since this is used in queries to the local app, I'd say this is rather an "N/A"